lib/src/model/settings/board_preferences.dart

```diff
@@ -132,6 +139,7 @@ class BoardPrefs with _$BoardPrefs implements Serializable {
     coordinates: true,
     pieceAnimation: true,
     showMaterialDifference: true,
+    clockPosition: ClockPosition.left,
```

Comment:
Shouldn't `right` be the default to match the current UI?

Reply:
Yes typo will fix

Reply:
Fixed